tasks/engine_setup.yml

```diff
@@ -44,7 +44,6 @@
     package:
       name: "*"
       state: latest
-      exclude: 'ansible'
     when: not ovirt_engine_setup_offline|bool
     tags:
       - "skip_ansible_lint"  # ANSIBLE0010
```

Comment:
Revert looks good, but we need to achieve the same functionality by passing `best=False` option to yum module:

```
- name: Copy yum configuration file
  copy:
    src: "/etc/yum.conf"
    dest: "/tmp/yum.conf"
    owner: root
    group: root
    mode: 0644
    remote_src: yes

- name: Set 'best' to false
  replace:
    path: "/tmp/yum.conf"
    regexp: '^best=True'
    replace: 'best=False'
    owner: root
    group: root
    mode: 0644

- name: Update all packages
    yum:
      name: '*'
      state: latest
      conf_file: /tmp/yum.conf
    when: not ovirt_engine_setup_offline|bool
    tags:
      - "skip_ansible_lint"  # ANSIBLE0010

- name: Remove temporary yum configuration file
  file:
    path: "/tmp/yum.conf"
    state: absent
  ignore_errors: True
```